front/src/applications/stdcmV2/components/StdcmOperationalPoint.tsx

```diff
@@ -131,9 +108,14 @@ const StdcmOperationalPoint = ({
     dispatchNewPoint(newPoint);
   };
 
-  const onSelectSuggestion = ({ value: suggestionName, uic }: { value: string; uic: number }) => {
-    setSearchTerm(suggestionName);
-    updateSelectedPoint(sortedSearchResults, uic);
+  const onSelectSuggestion = ({
+    value: suggestionName,
+    uic,
+  }: { value?: string; uic?: number } = {}) => {
+    setSearchTerm(suggestionName || '');
+    if (uic) {
+      updateSelectedPoint(sortedSearchResults, uic);
+    }
```

Comment:
Only a suggestion, but I find it clearer to manage the case where the selectedSuggestion is undefined like this (the types used are easier to understand too)

```suggestion
   const onSelectSuggestion = (selectedSuggestion?: Option) => {
    if (!selectedSuggestion) {
      setSearchTerm('');
      return;
    }
    const { value: suggestionName, uic } = selectedSuggestion;
    setSearchTerm(suggestionName);
    updateSelectedPoint(sortedSearchResults, uic);
```

Reply:
Done :)